### PR TITLE
doc: release: 3.3: Add working release notes file

### PR DESCRIPTION
**Pull request by nordicjm**
Adds a base working release notes file for the Zephyr 3.3 release.

### CODE REVIEW

**Review by nvlsianpu — DISMISSED** [review]

**Review by de-nordic — DISMISSED** [review]

File: doc/releases/release-notes-3.3.rst [hunk 1]
```diff
@@ -0,0 +1,213 @@
+:orphan:
+
+.. _zephyr_3.3:
+
+Zephyr 3.3.0 (Working Draft)
+############################
+
```
Comment (carlescufi) [inline doc/releases/release-notes-3.3.rst:7]: Would you mind adding the 

```
We are pleased to announce the release of Zephyr version 3.2.0.

Major enhancements with this release include:
```

section as shown in:
https://raw.githubusercontent.com/zephyrproject-rtos/zephyr/main/doc/releases/release-notes-3.2.rst

**Review by carlescufi — CHANGES_REQUESTED** [review]

File: doc/releases/release-notes-3.3.rst (hunk 1 above)
Comment (carlescufi) [reply doc/releases/release-notes-3.3.rst:7]: Otherwise people forget to add highlights. Thanks!

Comment (nordicjm) [reply doc/releases/release-notes-3.3.rst:7]: Fixed

**Review by de-nordic — APPROVED** [review]

**Comment by de-nordic** [comment]
@carlescufi Can we have the template in?

**Review by carlescufi — APPROVED** [review]

**Pull request merged**